CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Remove long-deprecated `perm*whitelist*` methods [#7401](https://github.com/hyperledger/besu/pull/7401)
 
 ### Additions and Improvements
+- Allow optional loading of `jemalloc` (if installed) by setting the environment variable `BESU_USING_JEMALLOC` to true/false. It that env is not set at all it will behave as if it is set to `true`
```

Comment:
This changelog entry needs to change specially if we simplify detectMalloc method. This can be rephrased similar to: "Simplified logging of jemalloc library at Besu startup".

Reply:
N/A